webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionsTableHelpers.js

```diff
@@ -76,13 +76,17 @@ export const buildTableRows = (groupedSubscriptions, availableQuantities, update
   return rows;
 };
 
-export const groupSubscriptionsByProductId = ({ results: subscriptions }) => {
+export const groupSubscriptionsByProductId = (
+  { results: subscriptions },
+  prevGroupedSubscriptions,
+) => {
   const grouped = {};
 
   subscriptions.forEach((subscription) => {
     if (grouped[subscription.product_id] === undefined) {
+      const prevOpenState = prevGroupedSubscriptions?.[subscription.product_id]?.open || false;
```

Comment:
Using JS optional chaining operator here (`?.`).  It's similar to Ruby's `&.` operator.  Based on this [canIuse entry](https://caniuse.com/#search=%3F.) confirming support in most modern browsers, and [this Babel PR](https://github.com/babel/babel/pull/10811) adding it to `@babel/preset-env` which will cover older browsers, I think it's safe to start using.  (more discussion in this Babel [issue](https://github.com/babel/babel/issues/10809))

Reply:
@sharvit @amirfefer @laviro  thoughts?

Reply:
looks like `foo?.bar` would transpile to
```js
foo === null || foo === void 0 ? void 0 : foo.bar;
```
Ugly but that's Babel's job anyway :smile:

Reply:
Curious to hear if there are other considerations, but seems fine from my perspective :+1: Thanks for checking!

Reply:
Looks like a great JS feature :fireworks: 
I would like to see it all over our code :)